OutOfSchool/OutOfSchool.AuthCommon/Services/CommonMinistryAdminService.cs

```diff
@@ -90,9 +88,9 @@ async Task<ResponseDto> CreateMinistryAdminOperation()
                     await transaction.RollbackAsync();
 
                     logger.LogError(
-                        $"Error happened while creation MinistryAdmin. Request(id): {requestId}" +
-                        $"User(id): {userId}" +
-                        $"{string.Join(Environment.NewLine, result.Errors.Select(e => e.Description))}");
+                        "Error happened while creation MinistryAdmin. User(id): {UserId}. {Error}",
+                        userId,
+                        string.Join(Environment.NewLine, result.Errors.Select(e => e.Description)));
```

Comment:
(Code style) The next code pattern 
```
string.Join(Environment.NewLine, result.Errors.Select(e => e.Description)))
```
seems to be duplicated in subsequent calls. Couldn't it be seprated as an extension method?

Reply:
Also same thing found in `ProviderAdminService`